android/src/test/java/com/amplitude/android/internal/gestures/AutocaptureGestureListenerClickTest.kt

```diff
@@ -145,12 +142,12 @@ class AutocaptureGestureListenerClickTest {
 
         verify(exactly = 1) {
             fixture.track(
-                ELEMENT_CLICKED,
+                "[Amplitude] Element Clicked",
```

Comment:
Why the switch to hardcorded strings in tests?

Reply:
This is just to make sure that we don't accidentally change the event/property name.